README.md

```diff
@@ -3,7 +3,9 @@
 ## About the scripts in this repository: 
 
 ### 1) Pre-processing pipeline: 
-For starting from scratch with an experiment ID or set of SRA's you would like to process into a counts gene matrix file.
+For starting from scratch with fastq files, an experiment ID or set of SRA's you would like to process into a counts gene matrix file.
+	### A) For smart-seq2 data or other single cell data that is avaiable on SRA
```

Comment:
```suggestion
	### A) For Smart-seq2 data or other single cell data that is available on SRA
```
What do you mean by?
> other single cell data 

This is also not rendering properly in the README. You probably need a new line before this one. Without a little bit more information, it's hard to tell what you mean by A and B here. I think your intention is that for Smart-seq2 data you would follow the steps labeled `A` below.

Reply:
Ah you have 
> and not droplet or tag-based

below (line 37).

Reply:
Made these changes, also went back and tried to be more consistent with calling this type of data "tag-based" instead of "10X" so that it includes both 10X and drop-seq technologies.